visualization.matrix/visualization.matrix.txt

```diff
@@ -0,0 +1 @@
+visualization.matrix https://github.com/xbmc/visualization.matrix master
```

Comment:
There should be a "Matrix" branch about, also seen on addon few parts where need cleaned up.

Mainly for here https://github.com/xbmc/visualization.matrix/blob/master/src/main.cpp#L32 as libXBMC_addon.h from old and since PVR system change no more present

Here https://github.com/xbmc/visualization.matrix/blob/master/visualization.matrix/addon.xml.in#L12 should the language name now "en_GB"

The to use only `#include <kodi/..>` (I prefer) or only `#include "kodi/.."` (not mixed)

Reply:
The plan was to use master for the moment and branch it off when the time arises. Sounds good?

As for the code related stuff, I would love to get a review of it. Other than that, I'll implement your suggestions.